pkgs/by-name/cu/cubeb/package.nix

```
@@ -2,85 +2,80 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  unstableGitUpdater,
   cmake,
   pkg-config,
   alsa-lib,
   jack2,
   libpulseaudio,
   sndio,
   speexdsp,
-  lazyLoad ? !stdenv.hostPlatform.isDarwin,
   alsaSupport ? !stdenv.hostPlatform.isDarwin,
   pulseSupport ? !stdenv.hostPlatform.isDarwin,
   jackSupport ? !stdenv.hostPlatform.isDarwin,
   sndioSupport ? !stdenv.hostPlatform.isDarwin,
   buildSharedLibs ? true,
 }:
 
-assert lib.assertMsg (
-  stdenv.hostPlatform.isDarwin -> !lazyLoad
-) "cubeb: lazyLoad is inert on Darwin";
-
-let
-  backendLibs =
-    lib.optional alsaSupport alsa-lib
-    ++ lib.optional jackSupport jack2
-    ++ lib.optional pulseSupport libpulseaudio
-    ++ lib.optional sndioSupport sndio;
-
-in
 stdenv.mkDerivation {
   pname = "cubeb";
-  version = "unstable-2022-10-18";
+  version = "0-unstable-2025-04-02";
 
   src = fetchFromGitHub {
     owner = "mozilla";
     repo = "cubeb";
-    rev = "27d2a102b0b75d9e49d43bc1ea516233fb87d778";
-    hash = "sha256-q+uz1dGU4LdlPogL1nwCR/KuOX4Oy3HhMdA6aJylBRk=";
+    rev = "975a727e5e308a04cfb9ecdf7ddaf1150ea3f733";
+    hash = "sha256-3IP++tdiJUwXR6t5mf/MkPd524K/LYESNMkQ8vy10jo=";
   };
 
+  outputs = [
+    "out"
+    "lib"
+    "dev"
+  ];
+
   nativeBuildInputs = [
     cmake
     pkg-config
   ];
 
-  buildInputs = [ speexdsp ] ++ lib.optionals (!stdenv.hostPlatform.isDarwin) backendLibs;
+  buildInputs =
+    [ speexdsp ]
+    ++ lib.optional alsaSupport alsa-lib
```

Dropping `backendLibs` makes sense given the changes in this PR, but I'd love to avoid the loss of domain knowledge. Maybe something like

```suggestion
    # These inputs are commonly lazy-loaded with dlopen, but this is not the case for this package.
    # If your package builds a vendored cubeb please make sure to include these.
    ++ lib.optional alsaSupport alsa-lib
```